server.js

```diff
@@ -12,7 +12,7 @@ var app = express();
 var port = process.env.PORT || 3000;
 
 /** this project needs a db !! **/ 
-// mongoose.connect(process.env.MONGOLAB_URI);
+// mongoose.connect(process.env.DB_URI);
```

Comment:
MLab users will have to migrate to Mongo Atlas in the next couple of months, so I went ahead and changed this to `DB_URI`. I'll also update this in the `.env` file on the Glitch boilerplate so users won't get an error.